src/lib/stringUtils.js

```diff
@@ -33,7 +33,7 @@ export function getFirstLineOfText(text) {
 }
 
 export function truncateTextWithEllipsis(str, length = 100, ending = '...') {
-  if (str.length > length) {
+  if (str && str.length > length) {
```

Comment:
Kan jo bruke `!!str`, så er den aldri være noe annet enn en boolean

Reply:
bare hvis lengde er > 100 (ref. length)

Reply:
Mente kun nullsjekken, er jo ikke akkurat nødvendig i javascript. Men liker å omforme objekt til faktisk boolean med double-not